### DISCUSSION TITLE
Mocking data with Percy CLI

### DISCUSSION THREAD

**Discussion by despotbg** [percy/cli#537] [Support] [answered]
Hi,
Is there an option to mock requests in Percy CLI like was the case with PercyScript? I don't see anything similar to setRequestInterception in Percy CLI tutorial?

**Comment by Robdel12** [comment, marked as answer]
Hey @despotbg! If you're looking to do request interception/mocking, using the Puppeteer SDK would be the right path: https://docs.percy.io/docs/puppeteer The PercyScript SDK was a thin wrapper around Puppeteer (a poor one -- we launched the browser and passed a single tab to use).
